catkit/hub/organize.py

```diff
@@ -89,7 +89,7 @@ def collect_structures(foldername, options):
                 filetype = ase.io.formats.filetype(posix_filename)
             except Exception as e:
                 continue
-            if filetype:
+            if filetype == 'traj':
```

Comment:
I couldn't make the test work for a general atoms format. The function tries to read everything in the folder.

Reply:
I had a more graceful handling of read error before. I put it back in, not sure how it disappeared. Maybe an unlucky merge somewhere. I think it is a crucial feature that we read arbitrary file as long as ASE can read them and not only *.traj file. Let's see if the tests pass now.